db/migrate/20151203230940_add_date_type_to_content_view_erratum_filter_rules.rb

```diff
@@ -0,0 +1,5 @@
+class AddDateTypeToContentViewErratumFilterRules < ActiveRecord::Migration
+  def change
+    add_column :katello_content_view_erratum_filter_rules, :date_type, :string, :default => "issued", :null => false
```

Comment:
@jlsherrill spoke to @mccun934 . He seems to be ok with issued being the default . I think its easy enough to change in the UI if they wanted updated rather than issued. Migration hack to achieve what you want seems not to be worth it IMO.